.github/workflows/cd-seed-chain.yml

```diff
@@ -76,8 +76,11 @@ jobs:
       - name: compile default erc20 contracts
         run: make compile-contracts
         working-directory: kava-bridge
+      - name: download seed script from master
```

Comment:
maybe we can checkout back to master after the kava install instead of downloading, in case we need other files later on?

Reply:
I like it, thats a better idea / more future proof

Reply:
ended up going with the original way since checking out the kava repo in the github action messed up the install paths for all the hardhat tools used by the seed script